server/sonar-server/src/main/java/org/sonar/server/rule/RegisterRules.java

```diff
@@ -114,6 +118,16 @@ public void start() {
     }
   }
 
+  private void persistRepositories(DbSession dbSession, List<RulesDefinition.Repository> repositories) {
+    dbClient.ruleRepositoryDao().truncate(dbSession);
```

Comment:
Do we really want to remove them all at each startup ?
Could be done later, but maybe we should only add new repos and remove no more existing ones.